DataFormats/L1TParticleFlow/src/HPSPFTau.cc

```diff
@@ -34,7 +33,7 @@ ostream& operator<<(ostream& os, const l1t::HPSPFTau& l1PFTau) {
   os << "seed:";
   if (l1PFTau.isChargedPFCandSeeded()) {
     os << " chargedPFCand";
-  } else if (l1PFTau.isPFJetSeeded()) {
+  } else if (l1PFTau.isJetSeeded()) {
     os << " PFJet";
```

Comment:
Calo?

Reply:
@sandeepbhowmik1 Can you check?

Reply:
yes it is calo jet now 
earlier we used PFJet

Reply:
Thanks.
Then please change `os << " PFJet"` to `os << " CaloJet"`;